package.json

```diff
@@ -70,7 +69,7 @@
     "@sentry/browser": "^5.0.5",
     "@sentry/cli": "^1.40.0",
     "electron": "^5.0.3",
-    "electron-builder": "^22.1.0",
+    "electron-builder": "^20.43.0",
```

Comment:
Downgraded this as newer versions have an error running Wine.

Reply:
This change came from another commit. But I think it is worth discussing.

Reply:
I think you upgraded it because the new macOS version requires it but in this version there is an error to generate Windows package (that's what I remember but I might be wrong). So in macOS version of @obiyankenobi notebook it's not needed.

Reply:
@pedroferreira1 is correct. This is the version that works for generating Windows builds. I remember you used to have this error @msbrogli . That's why I'm the only one that can generate packages nowadays.

What would happen is that I had to manually change it on `package.json` before generating the packages. Sometimes I'd forget and hit the error in the Windows build, which takes a while. So i thought it made more sense to downgrade ot.

Reply:
Wine stopped running when only 64 bit apps were allowed by macOS. Ok, seems safe to downgrade it.

Reply:
I also generate the `.pkg` file if we want to publish.